tests/Integ/S3EncryptionContext.php

```diff
@@ -93,7 +94,7 @@ public function iEncryptEachFixtureWith($wrapAlgorithm, $alias, $region, $cipher
         ]);
         $keyArn = $this->getKmsArnFromAlias($kmsClient, $alias);
 
-        $materialsProvider = new KmsMaterialsProvider(
```

Comment:
Should we be duplicating this and doing a test for v1 and a test for v2?

Or is V1 permanently deprecated